extensions/default/src/DicomWebDataSource/index.ts

```diff
@@ -22,7 +22,7 @@ const { DicomMetaDictionary, DicomDict } = dcmjs.data;
 const { naturalizeDataset, denaturalizeDataset } = DicomMetaDictionary;
 
 const ImplementationClassUID = '2.25.270695996825855179949881587723571202391.2.0.0';
-const ImplementationVersionName = 'OHIF-VIEWER-2.0.0';
+const ImplementationVersionName = 'OHIF-3.11.0';
```

Comment:
can you add comment what we changed this?

Reply:
It was larger than the maximum size that the DICOM field allows. Do you want me to add a comment in the code too?